.travis.yml

```diff
@@ -1,21 +1,21 @@
 sudo: false
 language: python
-python: "3.6"
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+  - "3.6"
+  - "3.7-dev"   # Change to "3.7" once Travis gets 3.7.0 on their machines
```

Comment:
This may be available as of last week: https://blog.travis-ci.com/2018-11-08-xenial-release:

> Python 2.7.15, 3.6.5, and 3.7.1.